src/Service/Users.php

```diff
@@ -79,8 +80,7 @@ public function isTokenValid($authToken, array $options = []) {
         ],
         'body' => $postBody,
       ];
-
-      $apiOptions = $this->config->get('isValidToken');
+      $apiOptions = $this->config['isValidToken'];
```

Comment:
Please can you rename `$this->config` to something a little more descriptive? Its not the entire "config", its only operations, therefore a more logical and descriptive name would be:
`$this->openamApiOperations`

Reply:
Updated.